.github/workflows/dbt-build.yml

```
@@ -0,0 +1,27 @@
+name: Docker image
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'warehouse/**'
+
+jobs:
+  build_push:
+    name: Package docker image
+    runs-on: ubuntu-18.04
+    steps:
+      #- uses: actions/checkout@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: warehouse
+          push: true
+          tags: ghcr.io/${{github.repository}}:latest
```

```suggestion
          tags: ghcr.io/${{github.repository}}/warehouse:latest
```

We should append a component name here so it can be pathed as a sibling alongside any other container images we publish from this repo

---

aha! can do